### DISCUSSION TITLE
How do you use IncludeTotalCount() on ITableQuery

### DISCUSSION THREAD

**Discussion by amertlich** [Azure/azure-mobile-apps#544] [Q&A]
I thought that I could do `.IncludeTotalCount().Take(0).ToAsyncEnumerable()` and the return type might be something I could cast to retrieve the count. 
At least this was how it was done preivously: [see stackoverflow answer here](https://stackoverflow.com/questions/35546443/how-to-access-the-data-returned-by-includetotalcount-with-imobileservicesynctabl)

How do we perform a count only query with the new client SDK?

**Comment by amertlich** [comment]
Additionally, using the select method on the IOfflineTable query doesn't appear to work as I would expect:
`.Select(t => new { t.Id}).Take(1).ToAsyncEnumerable();` throws an invalid operation that  "No 'id' properties found on type"
also `.Select(t => t.Id).Take(1).ToAsyncEnumerable();` does not work.

Also `.Take(0)` does not appear to be allowed

**Reply by adrianhall** [reply]
You make a good point - I don't have a mechanism for providing a total count.  Please file an issue.

As to the second issue, I'll have to write a test and see what is going on behind the scenes.  Please file an issue for that as well.

So - two new issues!
